src/test/java/api/InstancesApiExamples.java

```diff
@@ -873,22 +879,59 @@ public void cannotRemoveHRID() throws Exception {
     assertThat(existingInstance, is(createdInstance));
   }
 
+  @Test
+  public void canFrowardInstanceCreateFailureFromStorage() throws Exception {
+    final String expectedErrorMessage = "Instance-storage is temporary unavailable for create";
+
+    instancesStorageClient.emulateFailure(new EndpointFailureDescriptor()
+      .setFailureExpireDate(DateTime.now(DateTimeZone.UTC).plusSeconds(2).toDate())
+      .setBody(expectedErrorMessage)
+      .setContentType("plain/text")
+      .setStatusCode(500)
+      .setMethod(HttpMethod.POST));
```

Comment:
Personally, I'd statically import the method, as I think it makes this line more readable

```suggestion
      .setMethod(POST));
```